### PR TITLE
Add tv pages

### PR DESCRIPTION
**Pull request by Isak-Kallini**

### CODE REVIEW

**Comment by fnurkla** [comment]
image

**Comment by Isak-Kallini** [comment]
Wow yeah that looks terrible, how did I miss that 😆. Fixed now

File: src/routes/(app)/events/tv/+page@.svelte [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<script lang="ts">
+  import type { PageData } from "./$types";
+  import TvEventCard from "./TVEventCard.svelte";
+  export let data: PageData;
+</script>
+
+<section class="flex flex-col gap-2 px-14">
```
Comment (fnurkla) [inline src/routes/(app)/events/tv/+page@.svelte:7]:
```suggestion
<section class="flex flex-col gap-8 px-14">
```

Comment (fnurkla) [reply src/routes/(app)/events/tv/+page@.svelte:7]: Just to be consistent between news and events.

**Review by fnurkla — APPROVED** [review]
I think being consistent in the spacing would be nice, otherwise everything seems fine!

**Pull request merged**